### PR TITLE
Automatically generate image size versions that fit our agreed pattern

### PR DESCRIPTION
**Pull request by stevejalim**
## Summary

Please read #14578 for context.

This changeset eagerly generates the various sizes of image we _might_ use within CMS-managed pages, ready for use (and to avoid the web pod erroring when it can't generate a revision itself.)

Note that the eager generation is done synchronously in the POST request loop right now - this is fast/fine locally, but we will need to move it to an async worker task (via django-rq, which we'll need to add to things separately, before we can finish this work)

- [X] I used an AI to write some of this code: test skeleton, plus some simple regex tuning

## Issue / Bugzilla link

#14578 

## Testing

* `./manage.py createsuperuser` if you need to
* Log in at /cms-admin/
* Upload an image
* Look in bedrock/custom-images/ and see the various sizes have been pre-cut for you.

### CODE REVIEW

File: bedrock/cms/tests/test_template_markup.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import os
+import re
+from collections import defaultdict
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
```
[the remark this answers is not in the document]
Comment (stevejalim) [reply bedrock/cms/tests/test_template_markup.py:7]: Ooh yeah. I'll fix that

File: bedrock/cms/tests/test_template_markup.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    failures = defaultdict(list)
+
+    # now check each of them
+    for template_name in template_names:
+        with open(template_name) as fp:
+            html = fp.read()
+            matches = wagtail_jinja_image_tag_regex_pattern.findall(html)
+            for match in matches:
+                if match not in AUTOMATIC_RENDITION_FILTER_SPECS:
+                    failures[template_name].append(match)
```
[the remark this answers is not in the document]
Comment (stevejalim) [reply bedrock/cms/tests/test_template_markup.py:46]: Bad woah?

[the remark this answers is not in the document]
Comment (robhudson) [reply bedrock/cms/tests/test_template_markup.py:46]: haha, just reading it and realizing it reads all the templates checking for a regex pattern. Not a bad woah.

[the remark this answers is not in the document]
Comment (stevejalim) [reply bedrock/cms/tests/test_template_markup.py:52]: It's surprisingly fast - i thought it'd churn a bit, but it's a blink and done

**Pull request merged**